plugins/source/aws/resources/services/redshift/events.go

```diff
@@ -0,0 +1,52 @@
+package redshift
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/redshift"
+	"github.com/aws/aws-sdk-go-v2/service/redshift/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func Events() *schema.Table {
+	tableName := "aws_redshift_events"
+	return &schema.Table{
+		Name: tableName,
+		Description: `https://docs.aws.amazon.com/redshift/latest/APIReference/API_Event.html.
+
+Only events occurred in the last hour are returned.`,
```

Comment:
Hmm, this is quite a big caveat 😄 Is this an API limitation?

Reply:
Until we do stateful fetches it wouldn't be nice to pull all events (for the past 14 days, which is the api limit) again and again all the time. The api needs some time to go back (and an end time) with some source filtering options as well: https://docs.aws.amazon.com/redshift/latest/APIReference/API_DescribeEvents.html
one hour is the api default.